### DISCUSSION TITLE
Airtable import doesn't work?

### DISCUSSION THREAD

**Discussion by endolith** [AntlerVC/firetable#1343] [Support / Q&A] [answered]
I entered my API key and the Table URL with Base (app)/Table (tbl) but when I press "Continue" button it spins a second and then goes back to saying "Continue", with no error message or progress or anything.

https://docs.rowy.io/import-export-data/import-airtable

**Comment by BeeBombshell** [comment, marked as answer]
Hey @endolith, I checked and Airtable import works fine for me. Perhaps could you try to record the issue using https://www.jam.dev so that we can take a look?
